sys/select.go

```diff
@@ -1,8 +1,13 @@
-// +build !freebsd,!windows,!gccgo
+// +build !freebsd,!windows
 
 package sys
 
-import "syscall"
+import (
+	"reflect"
+	"syscall"
+)
+
+var nFdBits = (uint)(reflect.TypeOf(syscall.FdSet{}.Bits[0]).Size() * 8)
```

Comment:
You can use `unsafe.Sizeof`, which does not have the overhead of `reflect`. Also, in Go you do not need parentheses around the type name for casting. Use

```go
var nFdBits = uint(8 * unsafe.Sizeof(syscall.FdSet{}.Bits[0]))
```

Reply:
fixed.